src/style.css

```diff
@@ -1,3 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  position: relative;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+}
+
+.logo_link {
+  display: block;
+  width: 40px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list > :last-child {
+  margin-right: 0;
+}
+
+.nav_item {
+  margin-right: 20px;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  line-height: 15px;
+  padding: 23px 0;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.nav_link.is-active {
+  color: #00acdc;
+}
+
+.nav_link.is-active::after {
+  position: absolute;
+  bottom: 0;
+  content: "";
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
```

Comment:
Generally it's a good practice to keep all your colors as variables at the beggining of the file in the `:root` pseudoclass and later use these variables:
```css
:root {
  --light-blue: #00acdc;
}

...
background-color: var(--light-blue);
...
```
This is better for scalability of your app, it will be much easier to change your color scheme (or theme to dark for example) if all your colors are variables stored in one place. Also take notice of the naming of the variable I used here, don't use names like`main-color` but rather names that describe it like here `light-blue`